.template/addons/svgeez/.github/wiki/template.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+if Dir.exist?('.github/wiki')
+  use_source_path __dir__
+
+  copy_file 'Add-new-svg-icon.md', '.github/wiki/Add-new-svg-icon.md'
+
+  # SVG Sprite
+  insert_into_file '.github/wiki/_Sidebar.md', after: /## Operations.*\n/ do
+    <<~RUBY
+
```

Comment:
we can remove this empty line,  no?

Reply:
Thanks. Fixed in 1f5197abc6211ba7ac258c8fd69e339f7943486e

Reply:
Thanks. Removed in 1f5197abc6211ba7ac258c8fd69e339f7943486e